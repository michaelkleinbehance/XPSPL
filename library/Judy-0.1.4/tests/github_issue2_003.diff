002+ 
003+ Fatal error: Uncaught exception 'Exception' with message 'DateTime::__construct(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone.' in /Users/prggmr/Downloads/Judy-0.1.4/tests/github_issue2_003.php:7
002- k: 125, v: 2012
003- k: 521, v: 1983
004+ Stack trace:
005+ #0 /Users/prggmr/Downloads/Judy-0.1.4/tests/github_issue2_003.php(7): DateTime->__construct('2012-02-14')
006+ #1 {main}
007+   thrown in /Users/prggmr/Downloads/Judy-0.1.4/tests/github_issue2_003.php on line 7