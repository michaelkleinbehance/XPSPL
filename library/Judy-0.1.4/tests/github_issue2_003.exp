Set INT_TO_MIXED Judy object
k: 125, v: 2012
k: 521, v: 1983