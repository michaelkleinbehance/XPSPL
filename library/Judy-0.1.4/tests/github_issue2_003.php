<?php
/*
Ref. https://github.com/orieg/php-judy/issues/2
*/
$judy = new Judy(Judy::INT_TO_MIXED);
echo "Set INT_TO_MIXED Judy object\n";
$judy[125]  = new DateTime('2012-02-14');
$judy[521]  = new DateTime('1983-07-01');

foreach($judy as $k=>$v)
    print "k: $k, v: ".$v->format('Y')."\n";

unset($judy);
?>
