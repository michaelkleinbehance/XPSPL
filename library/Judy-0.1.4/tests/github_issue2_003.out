Set INT_TO_MIXED Judy object

Fatal error: Uncaught exception 'Exception' with message 'DateTime::__construct(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone.' in /Users/prggmr/Downloads/Judy-0.1.4/tests/github_issue2_003.php:7
Stack trace:
#0 /Users/prggmr/Downloads/Judy-0.1.4/tests/github_issue2_003.php(7): DateTime->__construct('2012-02-14')
#1 {main}
  thrown in /Users/prggmr/Downloads/Judy-0.1.4/tests/github_issue2_003.php on line 7